Octokit.Tests.Integration/Clients/RepositoryCommentsClientTests.cs

```diff
@@ -377,6 +378,29 @@ public TheCreateMethod()
             Assert.NotNull(retrieved);
         }
 
+        [IntegrationTest]
+        public async Task CanGetReactionPayload()
+        {
+            var commit = await SetupCommitForRepository(_github);
+
+            var comment = new NewCommitComment("test");
+
+            var result = await _github.Repository.Comment.Create(_context.Repository.Id,
+                commit.Sha, comment);
+
+            var reaction = await _reactionsClient.CommitComment.Create(_context.RepositoryOwner, _context.RepositoryName, result.Id, new NewReaction(ReactionType.Confused));
+            var retrieved = await _github.Repository.Comment.Get(_context.RepositoryOwner, _context.RepositoryName, result.Id);
+
+            Assert.True(retrieved.Id > 0);
+            Assert.Equal(1, retrieved.Reactions.TotalCount);
+            Assert.Equal(0, retrieved.Reactions.Plus1);
```

Comment:
Given we had issues with the Plus1 and Minus1 on the other reactions stuff, It'd be good to ensure our integration test covers deserializing all reaction summary fields.

How had would it be to have a unique value for each of the reactions, eg 5x Plus1's, 4x Minus1's, 3x Hooray, 2x Heart, 1x Laugh, 0x Confused ?